helm/templates/contour/crds/ingressroutes.yaml

```diff
@@ -0,0 +1,136 @@
+{{- if .Values.contour.enabled }}
```

Comment:
on EKS now you can use alb

Reply:
oh, yes I've  just found this announce https://aws.amazon.com/blogs/aws/new-application-load-balancer-support-for-end-to-end-http-2-and-grpc/ 
looks like for now it is possible, also it requires `aws-load-balancer-controller 2.0+` installed in kubernetes.
thank you!

Reply:
Ya I did set it up already on a personal
Account and works really well. I will be updating the eks manifests